nrf9160-hal/memory.x

```diff
@@ -1,28 +1,43 @@
-/* Linker script for the nRF9160 in Non-secure mode */
+/* Linker script for the nRF9160 in Non-secure mode. It assumes you have the
+Nordic Secure Partition Manager installed at the bottom of flash and that
+the SPM is set to boot a non-secure application from the FLASH origin below. */
+
 MEMORY
 {
-  /* SECURE_FLASH    : ORIGIN = 0x00000000, LENGTH = 256K */
-  /* NONSECURE_FLASH : ORIGIN = 0x00040000, LENGTH = 768K */
-  /* SECURE_RAM      : ORIGIN = 0x20000000, LENGTH = 64K */
-  /* LIBBSD_RAM      : ORIGIN = 0x20010000, LENGTH = 64K */
-  /* NONSECURE_RAM   : ORIGIN = 0x20020000, LENGTH = 128K */
-
-  FLASH : ORIGIN = 0x00040000, LENGTH = 768K
-  RAM : ORIGIN = 0x20020000, LENGTH = 128K
+	/*
+	 * This is where the Bootloader, Secure Partition Manager or
+	 * Trusted-Firmware-M lives.
+	 */
+	SECURE_FLASH : ORIGIN = 0x00000000, LENGTH = 256K
+	/*
+	 * This is where your non-secure Rust application lives. Note that SPM must agree this
+    * is where your application lives, or it will jump to garbage and crash the CPU.
```

Comment:
This line does not use a hard tab
```suggestion
	 * is where your application lives, or it will jump to garbage and crash the CPU.
```

That said, is this copied from Nordic, or should be rather use 4-space indentation like everywhere else here?

Reply:
Fixed.